src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorInitializeEndpoint.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CommonLanguageServerProtocol.Framework;
+using Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer
+{
+    [LanguageServerEndpoint(Methods.InitializeName)]
+    internal class RazorInitializeEndpoint : IRazorDocumentlessRequestHandler<InitializeParams, InitializeResult>
+    {
+        public bool MutatesSolutionState => true;
```

Comment:
```suggestion
        public bool MutatesSolutionState { get; } = true;
```

Reply:
Is this because the generated IL is more efficient as a getter?

Reply:
No, not in this case. In fact, in this case, the IL is actually more efficient than a getter-only property. In this case, the IL just pushes a constant on the stack and returns. However, a getter-only property has to read the backing field's value. So, the expression-bodied member is actually _slightly_ better, if you care about that level of micro-optimization (you shouldn't :smile:).

See my comment to David about for the reasoning why I'm a bit sensitive to these. In the future, I'll only call them out if they're actually problematic.